modular_skyrat/modules/assault_operatives/code/dynamic_rulsesets_roundstart.dm

```diff
@@ -0,0 +1,64 @@
+//////////////////////////////////////////////
+//                                          //
+//          ASSAULT OPERATIVES              //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/assault_operatives
+	name = "Assault Operatives"
+	antag_flag = ROLE_ASSAULT_OPERATIVE
+	antag_datum = /datum/antagonist/assault_operative
+	minimum_required_age = 14
+	restricted_roles = list(
+		JOB_CAPTAIN,
+		JOB_HEAD_OF_SECURITY,
+	)
```

Comment:
We probably want to add all the heads to that, to ensure there's as many heads to be against, as possible.